mods/ts/weapons/superweapons.yaml

```diff
@@ -56,19 +56,19 @@ SuicideBomb:
 	Warhead@2Res: DestroyResource
 
 IonCannon:
-	ValidTargets: Ground, Air
+	ValidTargets: Ground, Water, Air
```

Comment:
Water terrain does.
Sure, an unlikely scenario, but what  if two hover/amphibious actors are placed with some space between them so you want to shoot the ion cannon into that gap to kill both? Right now that would be impossible, because on water you can only target actors directly, not an empty water cell.